app/assets/javascripts/ui_models/application.ts

```diff
@@ -56,11 +56,9 @@ export class WebApplication extends SNApplication {
     $timeout: ng.ITimeoutService,
     scope: ng.IScope,
     onDeinit: (app: WebApplication) => void,
-    bridge: Bridge,
+    bridge: BrowserBridge,
```

Comment:
The reason we're relying on `Bridge` and not `BrowserBridge` is so different clients (namely Electron) can provide their own platform-specific implementations. If you need another property on `Bridge` you should add it there instead of depending on `BrowserBridge`